cfn-resources/testutil/testutil.go

```diff
@@ -1,26 +1,7 @@
 package testutil
 
-import (
-	"encoding/json"
-	"fmt"
```

Comment:
Lots of deletions in this file too, why?

Reply:
Lint errors because of unused testing 
code

Reply:
got it